orchagent/routeorch.cpp

```diff
@@ -466,6 +466,28 @@ void RouteOrch::doTask(Consumer& consumer)
             vector<string> ipv = tokenize(ips, ',');
             vector<string> alsv = tokenize(aliases, ',');
 
+            /*
+             * To check the validity of nexthop and make sure it works even in
+             * abnormal cases. One case is old format - empty "nexthop",
+             * then set ip to zero for backward compatibility.
```

Comment:
since you are handling two different cases here, please consider to have separate comments. the abnormal case, and back compatibility case. Please move the backware compability case before the for loop.

Reply:
All is for back compatibility including `resize() `, so it is difficult to separate cases, and I will separate `for` block and rewrite the comments.